templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -184,11 +206,6 @@ Resources:
         VPC: !GetAtt VPCStack.Outputs.VPCID
 
 Outputs:
-  DefaultKey:
-    Description: Default Ec2 keypair name
-    Value: !Ref 'KeyPairName'
-    Export:
-      Name: !Sub '${ExportPrefix}DefaultKey'
```

Comment:
Because `KeyPairName` is now optional and due to the limitations of CFN templating there is no way to check this value has been set in downsteam templates. Previously these dependent templates could rely on this value always being set because it was mandatory. 

As a result we remove this and the associated import from these downstream templates.